src/Smart.CFA.Pages/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+WORKDIR /app
+EXPOSE 5000
+
+ENV ASPNETCORE_URLS=http://+:5000
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["src/Smart.CFA.Pages/Smart.CFA.Pages.csproj", "src/Smart.CFA.Pages/"]
+RUN dotnet restore "src/Smart.CFA.Pages/Smart.CFA.Pages.csproj"
```

Comment:
I think a good practice is to only COPY the `.csproj` file, do the restore, then add the rest of the directory to build it. This allows to create a layer with the dependencies and avoid rebuilding it when the `.csproj` doesn't change.

Otherwise you can simplify the COPY line and simply copy the directory since the `.csproj` is in there.